xml/decoders/piko/cv897-1024_v5.1.xml

```diff
@@ -13,7 +13,7 @@
 <!-- for more details.                                                      -->
 <variables xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder.xsd">
 	<copyright xmlns="http://docbook.org/ns/docbook">
-		<year>2022</year>
+		<year>20234</year>
```

Comment:
This doesn't look right. Year 20234.

Reply:
sorry typing error

Reply:
No problem! That's why we have a review process.